__tests__/app.test.js

```diff
@@ -28,6 +29,47 @@ describe("GET: /api/topics", () => {
   });
 });
 
+describe("GET: /api/articles_id", () => {
+  it("200: GET responds with specific article object using article_id", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveLength(1);
```

Comment:
Great that you're checking the length to avoid false positives from being handed an empty array 🙌